app/src/main/kotlin/sync/Sync.kt

```diff
@@ -44,7 +45,7 @@ class Sync(
                                 id = place.getString("id"),
                                 lat = place.getDouble("lat"),
                                 lon = place.getDouble("lon"),
-                                tags = place.getJSONObject("tags"),
+                                tags = Tags(place.getString("tags")),
```

Comment:
I'm a bit uneasy about this, but it works it won't fail the syncing and following JSONObject implementation it seems it should work also.